server/server.go

```diff
@@ -410,7 +410,7 @@ func (s *server) newErrorEncoderWrapper() kithttp.ErrorEncoder {
 		s.errorEncoder(ctx, responseError, rw)
 
 		// Log the error and its stack. This is really useful for debugging.
-		s.logger.Log("level", "error", "message", "stop endpoint processing due to error", "stack", fmt.Sprintf("%#v", serverError))
+		s.logger.LogCtx(ctx, "level", "error", "message", "stop endpoint processing due to error", "stack", fmt.Sprintf("%#v", microerror.Mask(serverError)))
```

Comment:
`LogCtx` is useful, but additional masking doesn't bring more information. You have that in the `caller`.